ACPs/77-reinventing-subnets/README.md

```diff
@@ -213,167 +289,67 @@ An EVM Subnet may choose to implement this step like so:
 
-- `Balance` must be >= the greater of 5 $AVAX or two weeks of the current fee
+When it is known that a given `validationID` *is not and never will be* an existing validation period for a Subnet, the P-Chain must be willing to sign a `SubnetValidatorRegistrationMessage` for the `validationID` with `registered` set to `false`. This could be the case if the `expiry` time of the message has passed prior to the message being delivered in a `RegisterSubnetValidatorTx`, or if the Subnet Validator was successfully registered and then later removed. This enables the P-Chain to prove to validator managers that a validator has been removed or never added. The P-Chain must refuse to sign any `SubnetValidatorRegistrationMessage` where the `validationID` does not correspond to an active validator and the `expiry` is in the future.
 
-    This prevents Subnet Validators from being added with too low of an initial balance where they become immediately delinquent based on the continuous fee mechanism defined below. A Subnet Validator can leave at any time before the initial $AVAX is consumed and claim the remaining balance to the `ChangeOwner` defined in the transaction.
+#### `SetSubnetValidatorWeightTx`
 
-- `Signer` must correspond to the `blsPublicKey` defined in the warp message
-
-Note: There is no `EndTime` specified in this transaction. Subnet Validators are only removed when a `SetSubnetValidatorWeightTx` sets a validator's weight to `0`.
-
-### Modifying Subnet Validators
-
-#### SetSubnetValidatorWeightTx
-
-`SetSubnetValidatorWeightTx` is used to modify the voting weight of a Subnet Validator. For this transaction to be valid, 67% of the current Subnet Validator weight must sign the Warp message included in `Message`. Applications of this transaction could include:
-
-- Increase the voting weight of a Subnet Validator if a delegation is made on the Subnet
-- Increase the voting weight of a Subnet Validator if the stake amount is increased (by staking rewards for example)
-- Decrease the voting weight of a misbehaving Subnet Validator
-- Remove an inactive Subnet Validator
-
-Since there are no `EndTime`s enforced by the P-Chain, Subnets must use this transaction to set an expired validator's weight to `0`. When a validator's weight is set to `0`, they will be removed from the validator set. After the transaction is accepted, any $AVAX remaining in the `Balance` for the Subnet Validator being removed will be returned to the `ChangeOwner` defined in the `RegisterSubnetValidatorTx`.
+`SetSubnetValidatorWeightTx` is used to modify the voting weight of a Subnet Validator. The specification of this transaction is:
 
 ```golang
 type SetSubnetValidatorWeightTx struct {
     // Metadata, inputs and outputs
     BaseTx
-    // AddressedCall with Payload:
-    //   - ValidationID (SHA256 of the AddressedCall Payload of the RegisterSubnetValidatorTx adding the validator)
-    //   - Nonce
-    //   - Weight
+    // A SubnetValidatorWeightMessage payload
```

Comment:
This pattern is a _lot_ clearer than it was in the original spec. Thanks for taking the time to cleanup ❤️

Reply:
🫡